Sources/Controllers/RouteInfoMenu/OARouteLineAppearanceHudViewController.mm

```diff
@@ -985,6 +964,7 @@ - (void)clearColorSection:(BOOL)withTypes
         [self removeCellFromSection:sectionData cellKey:@"bottom_description"];
         [self removeCellFromSection:sectionData cellKey:@"gradientLegend"];
         [self removeCellFromSection:sectionData cellKey:@"paletteName"];
+        [self removeCellFromSection:sectionData cellKey:@"allColors"];
```

Comment:
NSArray *cellKeys = @[
    @"color_day_night_empty_space",
    @"color_day_night_value",
    @"color_grid",
    @"top_description",
    @"bottom_description",
    @"gradientLegend",
    @"paletteName"
];

for (NSString *key in cellKeys) {
    [self removeCellFromSection:sectionData cellKey:key];
}